provision-sm-md-lg/README.md

```diff
@@ -0,0 +1,81 @@
+# Provision VM Service Small Medium Large Example
+
+This example shows creating a provisioning dialog that modifies provisioning parameters
+
+## Architecture
+
+- `provision-vm-small-med-large.asl`: This workflow is run for every vm provisioned and is responsible for modifying the vm size. This will be used as a ManageIQ Service's Provisioning Entry Point.
```

Comment:
vm should be I think VM.

Reply:
- Do not use will.
- If vm is referred as first time here, use Virtual Machine (VM). For subsequent uses, use VM.

Might be better to re-phrase the sentence like this:

This workflow is provisioned for every provisioned Virtual Machine (VM), and is responsible for modifying the VM size. This is used as a ManageIQ provisioning service entry point.